site/vercel.json

```diff
@@ -1,4 +1,6 @@
 {
+  "installCommand": "cd .. && yarn install --ignore-engines",
+  "buildCommand": "cd .. && yarn build:base",
```

Comment:
just wanna make sure all of this runs from the root. I will look into updating the settings to always operate from root and correctly deploy the files from `site/dist` later